File: benchmark_template_v3/src/replay.ts
export {
  evaluateEntitlements,
  evaluateEntitlementsV2,
  reduceAccountState,
  reduceAccountStateV2,
  summarizeAccount,
  summarizeAccountV2
} from "./runtime.js";

export type {
  AccountState,
  AccountSummary,
  EntitlementState,
  NormalizedEvent
} from "./runtime.js";

File: benchmark_template_v3/ruleledger/replay.py
"""Replay, entitlement, and summary facade for RuleLedger v3."""

from __future__ import annotations

from ._runtime import (
    evaluate_entitlements,
    evaluate_entitlements_v2,
    reduce_account_state,
    reduce_account_state_v2,
    summarize_account,
    summarize_account_v2,
)

__all__ = [
    "evaluate_entitlements",
    "evaluate_entitlements_v2",
    "reduce_account_state",
    "reduce_account_state_v2",
    "summarize_account",
    "summarize_account_v2",
]
